diagnostic.md

```diff
@@ -57,13 +85,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$("li").parent("body").hide();
```

Comment:
`.parent()` only selects the immediate ancestor of the element that it's invoked on.  Here, `$("li").parent("body")` will return an `<ol>` element.